openam-documentation/openam-doc-source/src/main/docbkx/admin-guide/chap-troubleshooting.xml

```diff
@@ -83,13 +83,13 @@ $ jar cf ../openam.war *</screen>
     <answer>
      <para>OpenAM requires at least a maximum heap size of 1024 MB, with a 256
      MB maximum permanent generation heap size. For the Sun JVM, ensure the
-     container starts with <literal>-Xmx1024m -XX:MaxPermSize=256m</literal>
+     container starts with <literal>-Xmx1024m</literal>
      for these settings.</para>
      <para>If you do not know the settings used when the web container was
      started, use the <command>ps</command> command to check. In the following
      example, the web container is <literal>tomcat</literal>.</para>
      <screen>$ ps -ef | grep tomcat | grep Xm
-... -Xmx1024m -XX:MaxPermSize=256m ...</screen>
```

Comment:
these settings are still relevant when people are using 1.7. As long as 1.7 is the compiler source/target version, probably these should stick around.

Reply:
I'll see about parameterising that too